docs/install_from_github_release.md

```diff
@@ -4,13 +4,43 @@ This document explains how to install Element X Android from a Github Release.
 
 <!--- TOC -->
 
-* [Requirements](#requirements)
-* [Steps](#steps)
-* [I already have the application on my phone](#i-already-have-the-application-on-my-phone)
+* [Installing the universal APK](#installing-the-universal-apk)
+  * [Instructions](#instructions)
+  * [Steps](#steps)
+  * [I already have the application on my phone](#i-already-have-the-application-on-my-phone)
```

Comment:
This is added twice?

Reply:
Yes, the 2 paragraphs have the same name. the two links go to the first one :/ Not a big deal.

Reply:
Oh sorry, I didn't realise one was for the universal APK and another one for the app bundle 🙈 .